modules/jolt_physics/joints/jolt_cone_twist_joint_3d.h

```diff
@@ -50,8 +50,8 @@ class JoltConeTwistJoint3D final : public JoltJoint3D {
 	double swing_motor_target_speed_z = 0.0;
 	double twist_motor_target_speed = 0.0;
 
-	double swing_motor_max_torque = FLT_MAX;
-	double twist_motor_max_torque = FLT_MAX;
+	double swing_motor_max_torque = std::numeric_limits<double>::max();
+	double twist_motor_max_torque = std::numeric_limits<double>::max();
```

Comment:
These need to be `std::numeric_limits<float>::max()`, since they'll get cast to `float` when passed to Jolt, which would turn in to `infinity` with `std::numeric_limits<double>::max()`.

The fact that these members are `double` in the first place was a somewhat questionable decision on my part, as they at the very least should be `real_t`, but probably `float`, but that's better addressed in a separate PR.